bin/ovpn_run

```diff
@@ -35,6 +35,18 @@ function addArg {
     fi
 }
 
+function setupIptablesAndRouting {
```

Comment:
Can you add a comment above this and where it's called saying that it can be overridden by the `ovpn_env.sh` file so that the intent in is more clear?